File: SPENGO.Domain/Models/RequestModels/CommandRequestModels/UpdateWalletRequestModel.cs
using MediatR;
using SPENGO.Domain.Models.ResponseModel.CommandResponseModels;
using SPENGO.Domain.Models.ResponseModel.Shared;
using System;

namespace SPENGO.Domain.Models.RequestModels.CommandRequestModels
{
    public class UpdateWalletRequestModel : IRequest<ReponseModel<UpdateWalletResponseModel>>
    {
        public string Id { get; set; }
        public string Name { get; set; }
        public DateTime StartDate { get; set; }
        public DateTime EndDate { get; set; }
    }
}

File: Frontend/src/app/models/wallet/RequestModels/UpdateWalletRequestModel.ts
export default class UpdateWalletRequestModel {
    constructor(id: string, name: string, startDate: Date, endDate: Date) {
        this.id = id
        this.name = name
        this.startDate = startDate
        this.endDate = endDate
    }

    id: string
    name: string
    startDate: Date
    endDate: Date
}
